.github/workflows/reusable-build-test-config.yml

```
@@ -50,6 +50,12 @@ on:
         type: number
         default: 2
 
+      sanitizers:
+        description: "The sanitizers to enable ('Address,UndefinedBehavior' or 'Thread,UndefinedBehavior')."
```

```suggestion
        description: "The sanitizers to enable."
```

there are more possible values here, so I wouldn't add this comment